openexr-3.4.7/website/news.rst:284: WARNING: Duplicate explicit target name: "latestnewsend". [docutils]
--- openexr-3.4.7/website/news.rst.orig	2026-03-14 21:45:40.000000000 +0100
+++ openexr-3.4.7/website/news.rst	2026-03-21 17:55:42.963335962 +0100
@@ -281,8 +281,6 @@ Changes to the OpenEXR Python module:
 * 🐍 📦 ⚠️ `pypi` distributions now **add support
   for Python 3.13** and **drop support for Python 3.7**.
 
-.. _LatestNewsEnd:
-
 July 26, 2025 - OpenEXR 3.3.5 Released
 ======================================
 
